### ISSUE TITLE
Plugin does not check if WooCommerce is properly active

### ISSUE THREAD

**Issue by iamgabrielma** [woocommerce/woocommerce-gutenberg-products-block#300]
**Describe the bug**
Deactivating WooCommerce without deactivating woocommerce-gutenberg-products-block
 will cause fatals when calling to undefined functions from WooCommerce core.

**To Reproduce**

1. Deactivate WooCommerce having woocommerce-gutenberg-products-block
 active.
2. Check logs, you'll see that fails to call some core function like the following:

```
PHP Fatal error: Uncaught Error: Call to undefined function get_woocommerce_currency() in /wpcom-155025005/wp-content/plugins/woo-gutenberg-products-block/woocommerce-gutenberg-products-block.php:138 Stack trace: 
#0 /wordpress/core/5.0.2/wp-includes/class-wp-hook.php(286): wgpb_print_script_settings('') 
#1 /wordpress/core/5.0.2/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array) 
#2 /wordpress/core/5.0.2/wp-includes/plugin.php(453): WP_Hook->do_action(Array) 
#3 /wordpress/core/5.0.2/wp-admin/admin-footer.php(89): do_action('admin_print_foo...') 
#4 /wordpress/core/5.0.2/wp-admin/plugin-install.php(170): include('/wordpress/core...') 
#5 {main} thrown in /wpcom-155025005/wp-content/plugins/woo-gutenberg-products-block/woocommerce-gutenberg-products-block.php on line 138
```

**Expected behavior**
To not have access to the blocks, or to receive a warning that WooCommerce must be active.

**Comment by ryelle** [comment]
@iamgabrielma What version do you see this on, 1.2 or here on master?

**Comment by iamgabrielma** [comment]
Sorry for the delay @ryelle , this happened with `1.2`

**Comment by ryelle** [comment]
This has been fixed in master & will be released in 1.3 soon 👍 Going to close this for now, let me know if you see issues after updating to 1.3.
